### ISSUE TITLE
java2python

### ISSUE THREAD

**Issue by suresh40** [natural/java2python#36]
after  execution of "python setup.py install" , i tried to execute  j2py -- version

I am getting j2py is not recognised as internal or external command

**Comment by bcarroll** [comment]
Make sure the python scripts directory is in your PATH environment variable.  If you are on Windows, you might need to add .py to the j2py script for the OS to know it is a Python script.
.. or to be safe, make a copy of the j2py script.

Here is my setup on Windows 7:
<code>
set PATH=%PATH%;C:\Python27\Scripts
</code>

<code>
copy c:\Python27\Scripts\j2py c:\Python27\Scripts\j2py.py
</code>

**Comment by tusharsinghalrocks** [comment]
even after doing this still getting error 

![image](https://cloud.githubusercontent.com/assets/25664594/22786148/5a878bd6-eefd-11e6-81c1-ec9776eab064.png)

**Comment by samarth-29** [comment]
i am also getting the same error even after I copied j2py.py

**Comment by lion9** [comment]
File "C:\java2python\bin\j2py.py", line 113
    except (IOError, ), exc:

what may be the error?

**Comment by maifeeulasad** [comment]
Maybe the problem is , we have installed python and libraries at different `PATH`
So,
In the command prompt type "where python" first : 
![so](https://user-images.githubusercontent.com/29339330/52100233-ee37ca80-2600-11e9-8801-c8c3a89bfa00.PNG)
It will tell you , where your python is . Now instead of `C:\Python27\Scripts` , you can just write `python_path_without_exe` and followed by `Scripts`
See these for better understanding : 
![so1](https://user-images.githubusercontent.com/29339330/52100404-8cc42b80-2601-11e9-9eca-3047f38a6b87.PNG)
![so2](https://user-images.githubusercontent.com/29339330/52100464-ce54d680-2601-11e9-8585-fc59217c508c.PNG)

This worked for me.
ধন্যবাদ ।

**Comment by phs-sakshi** [comment]
even after this I am getting the same error. j2py is not recognized as an internal or external command.  Can you please help me out?

**Comment by maifeeulasad** [comment]
@B417037 we need more details, is python declared in path? How many version are you using at once ? etc.

**Comment by MinaTahaei** [comment]
> @B417037 we need more details, is python declared in path? How many versions are you using at once? etc.

I get the same error too. python is in my path, I only have 1 version but I still get this error

**Comment by phs-sakshi** [comment]
> @B417037 we need more details, is python declared in path? How many version are you using at once ? etc.

Yes, python is declared in the path. I am only using 1 version of python (python 2 as it does not work for python 3).

**Comment by me0wstic** [comment]
i'm getting the same error as well. python is declared in my path.

C:\Python27\Scripts>j2py --version
'j2py' is not recognized as an internal or external command,
operable program or batch file.

**Comment by raul-parada** [comment]
Did you solve this?

**Comment by maifeeulasad** [comment]
I don't have a Windows machine right now else I would make a video.

But you guys should follow this : https://www.computerhope.com/issues/ch000549.htm

In Windows you have to set the path variable. Then it'll work.
